online-store/src/index.ts

```diff
@@ -0,0 +1,51 @@
+import '../index.html';
+import '../styles.scss';
+import './components/loader/loader.scss';
+import './components/basket/basket';
+import { Button } from './components/button';
+import { loadProduct } from './components/cards/cards';
+import {
+  addLocalStoragePage,
+  getLocalStoragePage,
+  addLocalStorageBasket,
+  getLocalStorageBasket,
+} from './components/basket/localStorage';
+import { Routing, recoveryValue } from './components/routing/routing';
+import { SortProductCard } from './components/sort/sort';
+import { productsArrayI } from './components/types';
+import { renderBasket } from './components/basket/basket';
+
+loadProduct(50);
+
+Routing();
```

Comment:
it's better to stick to one approach to variable naming. Classes are usually named with a capital letter.